rq_dashboard/web.py

```diff
@@ -300,6 +309,9 @@ def job_view(job_id):
             rq_url_prefix="/",
             rq_dashboard_version=rq_dashboard_version,
             rq_version=rq_version,
+            deprecation_options_usage=current_app.config.get("DEPRECATED_OPTIONS")
```

Comment:
Just use `current_app.config.get("DEPRECATED_OPTIONS", False)`, no need for ternary operator here.